src/PKSim.Infrastructure/ProjectConverter/IObjectConverter.cs

```diff
@@ -7,16 +7,17 @@ namespace PKSim.Infrastructure.ProjectConverter
    public interface IObjectConverter : ISpecification<int>
    {
       /// <summary>
-      /// Should convert the object to convert and return the version corresponding to the new state of the object.
-      /// For example, if the converter converts from 5.0 to 5.1 and the current version is 5.2, it should return 5.1
+      /// Converss the <paramref name="objectToConvert"/> and returns the version corresponding to the new state of the object as well as a boolean indicating 
+      /// if a conversion actually took place for the <paramref name="objectToConvert"/>
+      /// For example, if the converter converts from 5.0 to 5.1 and the current version is 5.2, it should return 5.2
+      /// 
       /// </summary>
-      int Convert(object objectToConvert, int originalVersion); 
+      (int convertedToVersion, bool conversionHappened) Convert(object objectToConvert, int originalVersion);
```

Comment:
Same Idea as in core. We need to know if a conversion really happened or if no conversion was done for the converted object or xml